src/modules/error/ErrorCode.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import { FetchError, MappingError, ResponseError } from '@integreat-app/integreat-api-client'
+
+type ErrorCodesType = {|
+  PageNotFound: 'pageNotFound',
+  NetworkConnectionFailed: 'networkConnectionFailed',
+  NetworkRequestFailed: 'networkRequestFailed', // server's http-status was not 200
+  ResponseMappingFailed: 'responseMappingFailed',
+  UnknownError: 'unknownError'
+|}
+
+export const ErrorCodes: ErrorCodesType = {
```

Comment:
I think this file should be called `ErrorCodes` and should export this as default.